docs/csharp/language-reference/compiler-messages/cs8400.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Compiler Error CS8400
+ms.date: 07/11/2020
+f1_keywords:
+  - "CS8400"
+helpviewer_keywords:
+  - "CS8400"
+author: Youssef1313
+---
```

Comment:
I see other compile-messages have ms.assetid in the metadata. Is it required? What is it for? How to get the correct value for it?

Reply:
I believe that the `ms.assetid` is a carry over from old documents that were ported from MSDN days. It is not needed, nor is it applicable for new docs.